askama_parser/src/tests.rs

```diff
@@ -1216,13 +1218,6 @@ fn fuzzed_excessive_filter_block() {
         err.to_string().lines().next(),
         Some("your template code is too deeply nested, or the last expression is too complex"),
     );
-
-    let src = include!("../tests/fuzzed_excessive_filter_block.inc");
```

Comment:
Why was this test removed?

Reply:
See the initial message. It contained mostly invalid identifiers. The other test in the function tests the same problem.

Reply:
Shouldn't we update the test to have valid identifiers instead to continue checking the recursion limit. If I'm wrong please feel free to merge.

Reply:
The test file in 751_252 bytes long, and many of the identifiers contain invisible characters. I would not know how to best go about fixing that. That's why I simplified the test to only use identifier `x` in the first place. :)